compiler/rustc_parse/src/parser/item.rs

```diff
@@ -449,6 +450,21 @@ impl<'a> Parser<'a> {
         if end.is_doc_comment() {
             err.span_label(end.span, "this doc comment doesn't document anything");
         }
+        if end.meta_kind().is_some() {
+            let pos = end.span.hi() + BytePos(1);
```

Comment:
Does this cause an ICE with a multi-byte unicode character following the attribute? I fear that the `+ BytePos(1)` might.

You might want to instead use [`SourceMap::next_point`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_span/source_map/struct.SourceMap.html#method.next_point) or something similar.